web/toolbar.js

```diff
@@ -183,6 +194,7 @@ class Toolbar {
             isFromKeyboard: evt.detail === 0,
           });
         }
+        telemetry?.();
```

Comment:
... with the comment above implemented this would become:
```suggestion

        if (telemetry) {
          eventBus.dispatch("reporttelemetry", {
            source: this,
            details: telemetry,
          };
        }
```